app/customer.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict[str, int],
+            home_location: list[int, int],
```

Comment:
The type hint for `home_location` should be `list[int]` instead of `list[int, int]`. Lists in Python are not fixed in size, so specifying the number of elements in the type hint is not necessary.